main/main.cpp

```diff
@@ -1840,13 +1846,13 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::ARRAY, "rendering/gl_compatibility/force_angle_on_devices", PROPERTY_HINT_ARRAY_TYPE, vformat("%s/%s:%s", Variant::DICTIONARY, PROPERTY_HINT_NONE, String())), device_blocklist);
 	}
 
-	// Start with RenderingDevice-based backends.
+// Start with RenderingDevice-based backends.
```

Comment:
Which clang-format version do you have? It shouldn't do these changes.

Reply:
I tested with clang-format 15, it seems to be fine with both indentations. I think it makes more sense to keep at the code scope level.

Reply:
Done. I don't know what happened.